client/x/evmstaking/types/evmstaking.proto

```diff
@@ -32,3 +35,25 @@ message Withdrawal {
     (gogoproto.moretags) = "yaml:\"amount\""
   ];
 }
+
+// QueuedMessage is a message that can change the validator set and is delayed
+// to the end of an epoch
+message QueuedMessage {
+  // tx_id is the ID of the tx that contains the message
+  bytes tx_id = 1;
+  // msg_id is the original message ID, i.e., hash of the marshaled message
+  bytes msg_id = 2;
+  // block_height is the height when this msg is submitted to Babylon
```

Comment:
We should remove `Babylon` in the comments. Not just here but other places. 
Also if we reference their code, we should mention this in the README section of evmstaking module